src/.vuepress/components/ExtensionList.vue

```diff
@@ -1,51 +1,74 @@
 <template>
 	<div class="extension-list">
-		<div v-for="extensionGroup in extensions" :key="extensionGroup[0].lang">
-			<h3>
-				{{
-					extensionGroup[0].lang === "en"
-						? simpleLangName(extensionGroup[0].lang)
-						: langName(extensionGroup[0].lang)
-				}}
-				<span class="extensions-total">
-					Total:
-					<span class="extensions-total-sum">
-						{{ extensions.reduce((sum, item) => sum + item.length, 0) }}
+		<span class="filters-list">
+			<ElInput v-model="filters.search" placeholder="Search extensions by name..."></ElInput>
+
+			<ElSelect v-model="filters.lang" placeholder="Languages" multiple clearable>
+				<ElOption v-for="[group] in extensions" :key="group.lang" :value="group.lang"></ElOption>
+			</ElSelect>
+
+			<div>
+				Display extensions with NSFW content?
+				<ElRadioGroup v-model="filters.nsfw">
+					<ElRadioButton :label="true"></ElRadioButton>
+					<ElRadioButton label="all"></ElRadioButton>
+					<ElRadioButton :label="false"></ElRadioButton>
```

Comment:
Could this say Yes (true), No (false) and Don't care (all) instead. That would go better with the question
```suggestion
					<ElRadioButton label="Yes"></ElRadioButton>
					<ElRadioButton :label="No"></ElRadioButton>
					<ElRadioButton :label="Don't care"></ElRadioButton> 
```